packages/lib-core/src/types/plugin.ts

```diff
@@ -4,6 +4,7 @@ import type { PluginEntryModule } from './runtime';
 export type PluginRuntimeMetadata = {
   name: string;
   version: string;
+  apiAllowed?: Readonly<(string | undefined)[] | undefined>;
```

Comment:
@karelhala @florkbr 

Is this still up to date?

If so, I'd be very careful about adding new fields into the plugin manifest.

Can you please describe the purpose of this new field?

If our intent is to have individual plugins pass additional information to host applications like HAC, we should introduce some standard API to access such information.

For example, exposing the manifest via `PluginStore.getPluginInfo` method and corresponding React `usePluginInfo` hook.

Reply:
@vojtechszocs sorry for such a late reply, we've had to rearrange things slightly. Good poin on the custom field to allow any config from plugin to host app. I'll opene a new PR with this change (will probably close this 187 PR)